shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/sql/dml/select-or.xml

```diff
@@ -291,6 +291,26 @@
                 <table-factor>
                     <table name="t_order_item" alias="i" start-index="31" stop-index="42" />
                 </table-factor>
+                <joinSpecification>
```

Comment:
Where are these SQL Cases?

Reply:
In sharding-sql-test/src/main/resources/sql/supported/dml/select-or.xml